components/sup/src/manager/service/mod.rs

```diff
@@ -77,26 +81,31 @@ impl ServiceSpec {
             topology: Topology::default(),
             update_strategy: UpdateStrategy::default(),
             binds: vec![],
-            dev_config_from: None,
+            config_from: None,
```

Comment:
For a future discussion: I'm hoping to only use this (i.e. set `Some(…)`) through the `hab-sup start <PKG_IDENT_OR_ARTIFACT> …` invocation.